service/src/main/java/uk/gov/hmcts/reform/fpl/model/tasklist/TaskSection.java

```diff
@@ -0,0 +1,47 @@
+package uk.gov.hmcts.reform.fpl.model.tasklist;
+
+import java.util.List;
+import java.util.Optional;
+
+public class TaskSection {
+
+    private final String name;
+    private final List<Task> tasks;
+    private String hint;
+    private String info;
+
+    private TaskSection(String name, List<Task> tasks) {
+        this.name = name;
+        this.tasks = tasks;
+    }
+
+    public static TaskSection newSection(String name, List<Task> tasks) {
+        return new TaskSection(name, tasks);
+    }
+
+    public TaskSection withHint(String hint) {
+        this.hint = hint;
+        return this;
+    }
+
+    public TaskSection withInfo(String info) {
+        this.info = info;
+        return this;
+    }
+
+    public String getName() {
```

Comment:
If you wanted less boilerplate you could add an `@Getter` to the `name` and `tasks` variables

Reply:
done